src/pages/Onboard/AcceptInvitationPage.vue

```diff
@@ -0,0 +1,221 @@
+    <v-row v-if="invitationError" align="center">
+      <v-col class="grey--text text--lighten-5 mx-12">
+        <div class="display-1">
+          <span> {{ mutationErrorMessage || errorMessage }} </span>
+        </div>
+        <div>
+          <v-btn class="mt-8" color="primary" x-large @click="toDashboard">
+            Back to the dashboard!
+            <v-icon right>fas fa-rocket</v-icon>
+          </v-btn>
+        </div>
+      </v-col>
+    </v-row>
+    <v-row v-else align="center">
+      <v-col>
+        <div class="display-1 grey--text text--lighten-5">
+          You've been invited to join
+          <span class="font-weight-bold"> {{ teamName }}</span>
+        </div>
+        <div class="subtitle grey--text text--lighten-5 mt-4">
+          To accept, please click below.
+        </div>
+        <div class="mt-8">
+          <v-btn
+            class="mr-3"
+            color="accentPink"
+            dark
+            depressed
+            :loading="loading"
+            @click="accept"
+          >
+            <v-icon class="pr-4">fa-user-friends</v-icon>
+            Accept
+          </v-btn>
+          <v-dialog v-model="dialog" max-width="500">
+            <template #activator="{ on, attrs }">
+              <v-btn outlined class="white--text" v-bind="attrs" v-on="on">
+                No Thanks
+              </v-btn>
+            </template>
+
+            <v-card>
+              <v-card-title class="headline">
+                Are you sure you want to decline?
+              </v-card-title>
+
+              <v-card-text>
+                <div
+                  >Clicking <span class="font-weight-bold"> Decline </span> will
+                  delete your invitation and take you back to the dashboard. </div
+                ><div class="mt-2">
+                  If you don't want to confirm or delete your invitation right
+                  now, you can click on
+                  <span class="font-weight-bold"> Dashboard</span>. You'll be
+                  taken back to your team dashboard and be able to accept (or
+                  decline) the invitation later.
+                </div>
+              </v-card-text>
+
+              <v-card-actions>
+                <v-spacer></v-spacer>
+                <v-btn
+                  class="white--text"
+                  color="prefect"
+                  :loading="deleting"
+                  @click="decline"
+                >
+                  Decline
+                </v-btn>
+                <v-btn outlined color="prefect" @click="toDashboard">
+                  Dashboard</v-btn
+                >
+                <v-btn text color="prefect" @click="dialog = false"
+                  >Cancel</v-btn
+                >
+              </v-card-actions>
+            </v-card>
+          </v-dialog>
+        </div>
+      </v-col>
+    </v-row>
+    <v-row align="center">
+      <v-col>
+        <div class="ma-12 grey--text text--lighten-5 body-1 text--primary">
```

Comment:
Sorry, should have caught this before but can we make this text white? It's tough to see on the dark grey background.